staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/configmaplock.go

```diff
@@ -56,6 +58,16 @@ func (cml *ConfigMapLock) Get() (*LeaderElectionRecord, error) {
 			return nil, err
 		}
 	}
+	if cml.LeaseLock != nil {
+		leaseLock, err := cml.LeaseLock.Get()
```

Comment:
Why is this code here?  iirc this exists in the leader election code outside the resourcelock abstraction.  

/hold